src/main/java/tech/jhipster/lite/module/domain/gradleplugin/GradlePluginImport.java

```diff
@@ -0,0 +1,13 @@
+package tech.jhipster.lite.module.domain.gradleplugin;
+
+import tech.jhipster.lite.shared.error.domain.Assert;
+
+public record GradlePluginImport(String pluginImport) {
```

Comment:
How about this, since in reality the usage is not restricted to plugins:
```suggestion
public record BuildGradleImport(String import) {
```